src/Symfony/Component/Process/Tests/PhpProcessTest.php

```diff
@@ -26,4 +27,27 @@ public function testNonBlockingWorks()
         $process->wait();
         $this->assertEquals($expected, $process->getOutput());
     }
+
+    public function testCommandLine()
+    {
+        if (defined('PHP_BINARY')) {
+            $this->markTestSkipped('The PHP binary is easily available as of PHP 5.4');
+        }
+
+        $process = new PhpProcess(<<<PHP
+<?php echo 'foobar';
+PHP
+        );
+
+        $f = new PhpExecutableFinder();
+        $commandLine = $f->find();
+
+        $this->assertSame($commandLine, $process->getCommandLine(), '::getCommandLine() returns the command line of PHP');
+
+        $process->start();
+        $this->assertSame($commandLine, $process->getCommandLine(), '::getCommandLine() returns the command line of PHP');
+
+        $process->wait();
+        $this->assertSame($commandLine, $process->getCommandLine(), '::getCommandLine() returns the command line of PHP');
```

Comment:
the message should not be the same 3 times. It should change to describe the difference between these checks